src/content/en/tools/lighthouse/https.md

```diff
@@ -0,0 +1,49 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site is on HTTPS" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-19 #}
+{# wf_published_on: 2016-09-19 #}
+
+# Site is on HTTPS  {: .page-title }
+
+## Why the audit is important {: #why }
+
+All websites should be protected with HTTPS, even ones that don't handle
+sensitive data. HTTPS prevents intruders from tampering with or passively
+listening in on the communications between your site and your users.
+
+HTTPS is also a prerequisite for many new, powerful web platform features, such
+as taking pictures or recording audio.
+
```

Comment:
When we say why HTTPS is required, we frequently link to the policy. http://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features. We've done it on /web plenty of times.